node/config/config.go

```diff
@@ -190,10 +189,10 @@ func (c *Config) MakeSubscribeConfig() *config.Subscribe {
 }
 func (c *Config) MakeMinerConfig() *config.Producer {
```

Comment:
Rename `MakeMinerConfig` to `MakeVirtualSnapshotVerifierConfig`?

Reply:
The `virtual verifier` is just a parameter of the miner.